solidity/deploy/07_deploy_reimbursement_pool.ts

```diff
@@ -0,0 +1,38 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, ethers } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const staticGas = 40800 // gas amount consumed by the refund() + tx cost
+  const maxGasPrice = ethers.utils.parseUnits("500", "gwei")
+
+  if (deployments.getNetworkName() === "hardhat") {
+    const ReimbursementPool = await deploy("ReimbursementPool", {
+      contract: "ReimbursementPool",
+      from: deployer,
+      args: [staticGas, maxGasPrice],
+      log: true,
+    })
```

Comment:
After the recent @nkuba's changes, `ReimbursementPool` is only deployed once in `keep-core`.

Reply:
Removed the scripts around `ReimbursementPool` here [c4a4569](https://github.com/keep-network/tbtc-v2/pull/209/commits/c4a4569be2bad2a105d4b22f7b969318e903932c)